aws/cloudformation/alerting.yml.erb

```diff
@@ -1,13 +1,12 @@
-# Creates a SNS topic and Lambda which publish events to Honeybadger. This allows for any Cloudwatch Alarm
```

Comment:
I originally considered placing the resources in this template. These edits simply rearrange comments. This stack is updated manually, and will not be updated automatically upon deployment.